modules/improvedigitalBidAdapter.js

```diff
@@ -190,11 +190,15 @@ function getNormalizedBidRequest(bid) {
   if (keyValues) {
     normalizedBidRequest.keyValues = keyValues;
   }
-  if (localSize && localSize.w && localSize.h) {
+
+  if (config.getConfig('improvedigital.usePrebidSizes') === true && bid.sizes && bid.sizes.length > 0) {
+    normalizedBidRequest.format = bid.sizes;
```

Comment:
I don't think `sizes` is normally included in the `bid` object.  Is this some special case?

Reply:
It's supposed to be in the bid request, not response. Here's I believe where sizes field is populated: https://github.com/prebid/Prebid.js/blob/master/src/adapterManager.js#L98
Also, Appnexus adapter uses sizes the same way: https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L463

Reply:
Cool, I wasn't aware of that.  thx for the pointers.